shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/service/CarminDatasetPorcessingServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+package org.shanoir.ng.processing.carmin.service;
+
+import java.util.Optional;
+
+import org.shanoir.ng.processing.carmin.model.CarminDatasetProcessing;
+import org.shanoir.ng.processing.carmin.repository.CarminDatasetProcessingRepository;
+import org.shanoir.ng.shared.core.service.BasicEntityServiceImpl;
+import org.shanoir.ng.shared.event.ShanoirEvent;
+import org.shanoir.ng.shared.event.ShanoirEventService;
+import org.shanoir.ng.shared.event.ShanoirEventType;
+import org.shanoir.ng.shared.exception.EntityNotFoundException;
+import org.shanoir.ng.utils.KeycloakUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author KhalilKes
+ */
+@Service
+public class CarminDatasetPorcessingServiceImpl extends BasicEntityServiceImpl<CarminDatasetProcessing>  implements CarminDatasetProcessingService {
+
+        @Autowired
+        private CarminDatasetProcessingRepository carminDatasetProcessingRepository;
+
+        @Autowired
+        private ShanoirEventService eventService;
+
+        @Override
+        protected CarminDatasetProcessing updateValues(CarminDatasetProcessing from, CarminDatasetProcessing to) {
+                to.setIdentifier(from.getIdentifier());
+                to.setStatus(from.getStatus());
+                to.setName(from.getName());
+                to.setPipelineIdentifier(from.getPipelineIdentifier());
+                to.setStartDate(from.getStartDate());
+                to.setEndDate(from.getEndDate());
+                to.setTimeout(from.getTimeout());
+                to.setResultsLocation(from.getResultsLocation());
+
+                to.setDatasetProcessingType(from.getDatasetProcessingType());
+                to.setComment(from.getComment());
+                to.setInputDatasets(from.getInputDatasets());
+                to.setOutputDatasets(from.getOutputDatasets());
+                to.setProcessingDate(from.getProcessingDate());
+                to.setStudyId(from.getStudyId());
+
+                return to;
+        }
+
+        @Override
+        public CarminDatasetProcessing createCarminDatasetProcessing(final CarminDatasetProcessing carminDatasetProcessing) {
+                ShanoirEvent event = new ShanoirEvent(ShanoirEventType.IMPORT_DATASET_EVENT,
```

Comment:
I am not sure to require an event here, as we do not do any import here, we only save a json object to the database and do not process any images or files

Reply:
probably all the event code could be removed here please

Reply:
Okay, will do

Reply:
Perfect, thx!